### DISCUSSION TITLE
Dark-mode-img-color-conversion

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#4209] [Show and tell]
If you use dark theme or use hightlights_list, you may need this widget. This script is universal, No matter which theme you use.
https://github.com/SiriusXT/trilium-theme-blue/blob/main/Dark-mode-img-color-conversion.js
1. Under the black theme, the white picture may be too conspicuous. This plugin inverts white image colors to fit them into a dark theme.
2. The background color of the text may be too bright, destroying the beauty of the theme. This plugin sets the text background color to be semi-transparent.

**Original style:**
![图片](https://github.com/zadam/trilium/assets/37627919/991a60ee-f397-4383-bf96-a94e393c23dd)

**New style:**
![图片](https://github.com/zadam/trilium/assets/37627919/af584a2d-d93e-425b-b096-48b15cb8201c)

**Comment by HilkopterBob** [comment]
please encaplsulate your code so we do not need to use your theme.

**Reply by SiriusXT** [reply]
You don't need to use this theme, just add the widget tag to the js script directly. This script is universal.

**Comment by SunriseWind** [comment]
Is there a solution to do the same thing in Book -> Calendar type notes?

https://github.com/TriliumNext/Trilium/issues/7569

<img width="475" height="277" alt="image" src="https://github.com/user-attachments/assets/6264656e-0031-4807-9c94-5d32b86f9ca6" />
